### PR TITLE
added custom popup and icon

### PR DESCRIPTION
**Pull request by giuliaghisini**
Added custom popup and icon for maker

### CODE REVIEW

**Comment by nzambello** [comment]
@giuliaghisini could you add the new props in README docs?
Could you explain what clusterOptions can be used for or a reference to its documentation?

**Comment by giuliaghisini** [comment]
> @giuliaghisini could you add the new props in README docs?
> Could you explain what clusterOptions can be used for or a reference to its documentation?

done! i've removed clusterOptions because it was unused and it was a typo from previous implementations

File: src/components/OSMMap/OSMMap.css [hunk 1]
```diff
@@ -29,3 +29,22 @@
 #geocoded-result {
   height: 300px;
 }
+
+.marker-cluster {
+  background-color: #ff8d00;
+  border-radius: 100%;
+  box-shadow: 0px 0px 8px 2px #ff8d00;
```
Comment (nzambello) [inline src/components/OSMMap/OSMMap.css:37]:
```suggestion
  -webkit-box-shadow: 0px 0px 8px 2px #ff8d00;
  box-shadow: 0px 0px 8px 2px #ff8d00;
```

File: src/components/OSMMap/OSMMap.css [hunk 2]
```diff
@@ -29,3 +29,22 @@
...
+
+.marker-cluster div {
+  width: 30px;
+  height: 30px;
+  margin-left: 5px;
+  margin-top: 5px;
+  text-align: center;
+  border-radius: 15px;
+  font-size: 12px;
+}
```
Comment (nzambello) [inline src/components/OSMMap/OSMMap.css:48]:
```suggestion
}

```

File: src/components/OSMMap/OSMMap.jsx [hunk 3]
```diff
@@ -26,11 +27,38 @@ const OSMMap = ({
...
+          )}
+          {showPopup && position.popupContent && (
+            <Popup offset={[0, -22]} direction="top">
+              {position.popupContent}
+            </Popup>
+          )}
+        </Marker>
+      ))}
+    </>
+  );
```
Comment (nzambello) [inline src/components/OSMMap/OSMMap.jsx:61]:
```suggestion
  );
  
```

**Review by nzambello — APPROVED** [review]

**Pull request merged**